extendr-api/src/robj_ndarray.rs

```diff
@@ -1,3 +1,57 @@
+//! Defines conversions between R objects and the [ndarray](https://docs.rs/ndarray/latest/ndarray/) crate, which offers native Rust array types and numerical computation routines.
+//!
+//! To enable these conversions, you must first enable the `ndarray` feature for extendr:
+//! ```toml
+//! [dependencies]
+//! extendr-api = { version = "0.3.1", features = ["ndarray"] }
+//! ```
+//!
+//! Specifically, extendr supports the following conversions:
+//! * [`Robj` → `ArrayView1`](FromRobj#impl-FromRobj<%27a>-for-ArrayView1<%27a%2C%20T>), for when you have an R vector that you want to analyse in Rust:
```

Comment:
In #443 I am trying to add support for `TryFrom<Robj>`, which is the proper way to handle conversions. Should we document it instead of obsolete `FromRobj` or wait till I merge my PR?

Reply:
For the sake of simplicity, maybe we just leave my PR as is, and then we can update the link in your PR? Unless yours gets merged first, in which case I'll do the fix here.